### ISSUE TITLE
f-droid release?

### ISSUE THREAD

**Issue by winst0nsmith** [thunderbird/thunderbird-android#8431]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

8.0b4

### Problem you are trying to solve

unable to install thunderbird-android from non-google app stores.

### Suggested solution

when can we expect a thunderbird-android build in fdroid?

k-9's been there forever, and the only info i can find on this seems to indicate that it isn't possible to have both k-9 and thunderbird in the same repo? is this really that big a deal?

can't stand google play, sorry.

**Comment by kewisch** [comment]
Yes, we'll have an f-droid release.  We'd like to use reproducible builds, and I'm working on getting that set up. If you don't mind I'm going to close this issue since this is actively on my radar and I might otherwise forget to close it.

**Comment by asandikci** [comment]
Thanks for the work!
(more info: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/16035)

**Comment by Harrier-Du-Bois** [comment]
Apparently there's been an update which is available via google play store but hasn't yet cropped up on fdroid.
